templates/slides/motion.html

```diff
@@ -90,7 +84,7 @@ <h2 class="projector_h2">
     <section id="text-wrapper">
       <div id="text" style="{{ if .ShowSidebox }}width: calc(100% - 250px);{{ end }}">
         <!-- Text -->
-        {{ if .Motion.Text }}
+        {{ if .Preamble }}
```

Comment:
Maybe better to not display it if no text?

```suggestion
        {{ if and .Preamble .Motion.Text }}
```